### PR TITLE
feat(checkout): CHECKOUT-7036 Filter out convertcart-related issues on Sentry.

### PR DESCRIPTION
**Pull request by bc-peng**
## What?
Filter out convertcart-related issues on Sentry.

## Why? [CHECKOUT-7036](https://bigcommercecloud.atlassian.net/browse/CHECKOUT-7036)
Reduce the amount of noisy errors we receive, such as [3505225940](https://sentry.io/organizations/bigcommerce/issues/3505225940/?project=1542560).

## Testing / Proof
- Added one unit test.
- CI

### CODE REVIEW

File: packages/core/src/app/common/error/SentryErrorLogger.ts [hunk 1]
```diff
@@ -135,6 +135,12 @@ export default class SentryErrorLogger implements ErrorLogger {
     }
 
     private handleBeforeSend: (event: Event, hint?: EventHint) => Event | null = (event, hint) => {
+        if (
+            event.breadcrumbs?.filter((breadcrumb) => breadcrumb.data?.url.includes('convertcart'))
```
Comment (animesh1987) [inline packages/core/src/app/common/error/SentryErrorLogger.ts:139]: Do you for see us having a list of works like `convertcart` that we need to ignore in future?

Comment (bc-peng) [reply packages/core/src/app/common/error/SentryErrorLogger.ts:139]: For other issues, we might have to use other signatures to identify them.
Right now, I think we just aim to remove `convertcart`.

Comment (snaderiBC) [inline packages/core/src/app/common/error/SentryErrorLogger.ts:139]: Is it possible to use and take advantage of https://docs.sentry.io/platforms/javascript/guides/capacitor/configuration/options/#deny-urls

Comment (bc-peng) [reply packages/core/src/app/common/error/SentryErrorLogger.ts:139]: Yesterday, the same Sentry issue's URL was https://www.extremekartz.com/checkout, today it is https://mcneelamusic.com/checkout/order-confirmation.
Therefore, I think it is better to identify `convertcart` in breadcrumbs.

Comment (animesh1987) [reply packages/core/src/app/common/error/SentryErrorLogger.ts:139]: Found this in github issues https://github.com/getsentry/sentry-javascript/issues/3147, seems similar. It also links a PR for denyUrls.

Comment (bc-peng) [reply packages/core/src/app/common/error/SentryErrorLogger.ts:139]: I have added `convertcart` to `denyUrls`.
Maybe we can keep the filter in `beforeSend` as it is the simlar solution mentioned in https://github.com/getsentry/sentry-javascript/issues/3147 .

**Review by animesh1987 — APPROVED** [review]

**Pull request merged**